control-plane/src/health.rs

```diff
@@ -1,19 +1,35 @@
 use crate::enclave_connection::get_connection_to_enclave;
 use crate::error::ServerError;
 use hyper::{Body, Request, Response};
+use lazy_static::lazy_static;
 use serde::{Deserialize, Serialize};
 use shared::server::health::{HealthCheckLog, HealthCheckStatus};
 use shared::server::{error::ServerResult, tcp::TcpServer, Listener};
 use shared::{env_var_present_and_true, ENCLAVE_HEALTH_CHECK_PORT};
 use std::net::SocketAddr;
+use tokio::sync::Mutex;
+
+lazy_static! {
```

Comment:
can use a oncecell here instead, it's been added to std https://doc.rust-lang.org/std/cell/struct.OnceCell.html

Reply:
Didn't know it was in std, thought it was still a crate! Used OnceLock instead so it's thread safe!